docs/getting-started.md

```diff
@@ -9,65 +9,234 @@ sidebar_label: Getting started
+
+How does Aragon fit in? Aragon provides you with tools (apps) that make it easy for you to create DAOs -- you can think of Aragon apps as lego bricks that can be flexibly combined to make DAOs (the final lego structures).
+
+Another way to think of Aragon is as a new operating system (OS) for setting up organizations. One that is far more efficient than the existing OS (lawyers, administrative headaches, etc) we are used to dealing with.
+
+### More on DAOs
+
+For those of you who are new to DAOs, in this section we'll briefly cover the sorts of things DAOs can enable, when you might want to use a DAO, as well as the trends around the world accelerating their adoption.
+
+#### DAOs can enable: 
+
+- Shared bank accounts with custom rules and permissions
+- Payroll that automatically runs by itself
+- Built-in voting on important topics
+
+#### DAO use cases: 
+
+- Part-time projects with friends or strangers
+- Future of work: people working part time on multiple things for short periods of time
+- Temporary pop-up companies
+- Companies in authoritarian jurisdictions
+- Global, distributed teams
+
+#### Trends around the world accelerating DAO adoption: 
+
+- The rise of populist authoritarians
+- The future of work (remote and distributed)
+- The growing de-platforming problem
+- The rise of decentralized finance
+- The [increasing](https://medium.com/complex-systems-channel/teams-a-manifesto-7490eab144fa) [complexity](https://necsi.edu/complexity-rising-from-human-beings-to-human-civilization-a-complexity-profile) of human civilization
+
+
+
+### Web3
+
+Unless you've been living under a rock for the last few years, you've probably come across the term web3 😋. But what does it mean exactly? And why do we care about it in the context of Aragon?
+
+Web3 is the vision of a fully decentralized web. One of the craziest things to wrap your head around is that in web3, apps don't need a central server to fetch data from!
+
+How is this possible? In a nutshell, thanks to something called peer-to-peer data architectures. The key point is that in a P2P architecture, instead of requesting data from a central server, you request it from multiple computers (peers) around you.
+
+While this is nothing new in itself -- P2P architectures have existed since the 1990’s (where they rose to fame with file sharing programs like BitTorrent and Napster) -- what's new is the addition of cryptography and economic incentives to these architectures.
 
-What you need to get started building with Aragon:
+This fusion of these seemingly disparate disciplines was the big innovation behind Bitcoin, and has since led to the emergence of a new field of research devoted to their intersection (what we now call cryptoeconomics). 
```

Comment:
```suggestion
The fusion of these seemingly disparate disciplines was the big innovation behind Bitcoin, and has since led to the emergence of a new field of research devoted to their intersection (what we now call cryptoeconomics). 
```